aitelemetry/telemetrywrapper.go

```diff
@@ -15,6 +15,7 @@ const (
 	resourceGroupStr                 = "ResourceGroup"
 	vmSizeStr                        = "VMSize"
 	osVersionStr                     = "OSVersion"
+	osStr                            = "OS"
```

Comment:
are u using this? if not remove it

Reply:
Yes I am using it. I am adding it as part of properties.